src/main/resources/hudson/plugins/warnings/GroovyParser/config.jelly

```diff
@@ -1,27 +1,27 @@
 <?jelly escape-by-default='true'?>
```

Comment:
This line had a unix EOL code (\n) at its end.
All the other lines in this file had MSDOS EOL codes (\r\n) at their ends.
...except the last line, which had no EOL code.